src/widgets/histogram/chart.js

```diff
@@ -1499,6 +1500,9 @@ module.exports = cdb.core.View.extend({
       })
       .attr('y', self.chartHeight())
       .attr('height', 0)
+      .attr('data-tooltip', function (d) {
```

Comment:
Not a big fan of one-letter function arguments, could we use `data` or `tooltipData` or something more explicit?

Reply:
This is really a convention from d3, I see your point but two reasons why I won't fix:
- The "parent" of that `d` attribute is `data` so it would be really confusing. Also, that `d` depends on the use you are doing of that value. Some times it would be related to the tooltip, other to axis, labels or something else.
- There are a lot of places in the code to be changed, names to get addressed... I get you but it's out of scope.

Reply:
👍